### PR TITLE
Updating pytest for sflow

### PR DESCRIPTION
**Pull request by dgsudharsan**
**What I did**
Update pytests for sflow

**Why I did it**
The updated pytests test the flow from cfg_db to sflowmgr to sfloworch to asic_db. Thus end to end flow is tested

**How I verified it**
Execute the tests

**Details if related**
DO NOT MERGE UNTIL THE BELOW PULL REQUEST IS MERGED
https://github.com/Azure/sonic-buildimage/pull/3595

### CODE REVIEW

**Comment by dgsudharsan** [comment]
sudo pytest -s -v --dvsname=vs test_sflow.py
========================================================= test session starts =========================================================
platform linux2 -- Python 2.7.15rc1, pytest-3.3.0, py-1.8.0, pluggy-0.6.0 -- /usr/bin/python
cachedir: .cache
rootdir: /home/sudharsan/pytest, inifile:
collected 6 items

test_sflow.py::TestSflow::test_defaultGlobal remove extra link dummy
PASSED                                                                             [ 16%]
test_sflow.py::TestSflow::test_globalAll PASSED                                                                                 [ 33%]
test_sflow.py::TestSflow::test_InterfaceSet PASSED                                                                              [ 50%]
test_sflow.py::TestSflow::test_defaultRate PASSED                                                                               [ 66%]
test_sflow.py::TestSflow::test_ConfigDel PASSED                                                                                 [ 83%]
test_sflow.py::TestSflow::test_Teardown PASSED                                                                                  [100%]
====================================================== 6 passed in 71.33 seconds ======================================================

**Comment by prsunny** [comment]
@dgsudharsan , can you fix the conflict?

**Comment by dgsudharsan** [comment]
retest vs please

**Comment by dgsudharsan** [comment]
retest this please

**Comment by dgsudharsan** [comment]
retest this please

File: tests/test_sflow.py [hunk 1]
```diff
@@ -96,19 +237,19 @@ def test_InterfaceSet(self, dvs, testlog):
...
-                assert fv[1] == "1000"
+                assert fv[1] == rate
 
-        ptbl._del("Ethernet0")
+        ctbl._del("Ethernet4")
 
     def test_ConfigDel(self, dvs, testlog):
         self.setup_sflow(dvs)
-        ptbl = swsscommon.ProducerStateTable(self.pdb, "SFLOW_SESSION_TABLE")
+        ctbl = swsscommon.Table(self.cdb, "SFLOW_SESSION_TABLE")
```
Comment (wendani) [inline tests/test_sflow.py:246]: schema for config_db is `SFLOW_SESSION` not  `SFLOW_SESSION_TABLE`

**Pull request merged**